go.mod

```diff
@@ -200,6 +200,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/klauspost/compress v1.15.15
```

Comment:
Can this be put in the other require section above and re-run `go mod tidy` so we don't have too many require sections?